source/tyr/tyr/resources.py

```diff
@@ -998,6 +999,7 @@ def post(self):
             autocomplete_parameter.admin = args['admin']
             db.session.add(autocomplete_parameter)
             db.session.commit()
+            create_autocomplete_depot(autocomplete_parameter.name)
```

Comment:
for an asynchronous call you need to use the ```delay()``` method: http://docs.celeryproject.org/en/latest/userguide/calling.html